pymc4/flow/executor.py

```diff
@@ -276,7 +277,7 @@ def evaluate_model(
                     elif isinstance(dist, distribution.Distribution):
                         try:
                             return_value, state = self.proceed_distribution(
-                                dist, state, sample_shape=sample_shape, num_chains=num_chains,
+                                dist, state, sample_shape=sample_shape, draws=draws,
```

Comment:
I know it will be inconsistent with PyMC3, but can you rename draws to particles or replicas?
FYI @aloctavodia

Reply:
I've changed it to replicas. Will wait to @aloctavodia  though.